chia/wallet/wallet_block_store.py

```diff
@@ -62,6 +65,9 @@ async def create(cls, db_wrapper: DBWrapper):
         # Height index so we can look up in order of height for sync purposes
         await self.db.execute("CREATE INDEX IF NOT EXISTS height on block_records(height)")
 
+        # earlier versions of chia created this index despite no lookups needing
+        # it. For now, just don't create it for new installs. In the future we
+        # may remove the index from existing installations as well
         # Do not create named index on column header_hash, because it already is auto indexed as of Primary Key (Index Name sqlite_autoindex_block_records_1)
         #await self.db.execute("CREATE INDEX IF NOT EXISTS hh on block_records(header_hash)")
```

Comment:
I think you should remove this commented-out code as well. the comment ought to be enough to prevent us from adding it back in the future

Reply:
Ok. You mean I should just use the phrase you used before? And also remove the CREATE INDEX and not just comment it out?

Reply:
The conservative approach we've taken with the other redundant indices was to stop creating them, but not actively drop them from users (yet). The idea being that users are more likely to downgrade one or two releases than much more than that. If we introduce this change now, and 3 releases from now introduce actively removing the index, it's less likely to cause issues for users.

in that approach, we leave a commented-out command to `DROP INDEX IF EXISTS` here, that we un-comment in the future.